lambda_function.py

```diff
@@ -80,16 +80,35 @@ def elasticsearch_handler(event, context):
     print(f"Successfully processed {success}/{total} items for opensearch")
 
 
+def _send_to_splunk(events, errors, splunk_hec_url, splunk_hec_token):
+    try:
+        response = requests.post(
+            splunk_hec_url,
+            headers={'Authorization': f'Splunk {splunk_hec_token}'},
+            json=events,
+            timeout=30
+        )
+        response.raise_for_status()
+        return len(events)
+    except Exception as e:
+        errors.append(str(e))
```

Comment:
You can print out the error here 
```suggestion
        logger.error(str(e))
```